services/brig/test/integration/API/Search.hs

```diff
@@ -54,6 +54,11 @@ tests opts mgr galley brig = do
     testGroup "search" $
       [ testWithBothIndices opts mgr "by-name" $ testSearchByName brig,
         testWithBothIndices opts mgr "by-handle" $ testSearchByHandle brig,
+        testGroup "ASDFASDF search by email" $
+          [ testWithBothIndices opts mgr "private account" $ testSearchByEmailPrivate brig,
+            testWithBothIndices opts mgr "different team" $ testSearchByEmailDifferentTeam brig,
+            testWithBothIndices opts mgr "same team" $ testSearchByEmailSameTeam brig
+          ],
```

Comment:
They need to be changed to call the new end-point, and may not fully make sense any more they way they work now.